### PR TITLE
Add support for ghost invocies

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Added support for ghost invocies to invoice command

### CODE REVIEW

File: offchain/sign_ghost_payment_request.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+            if (args.id === forward.hash) {
+              forward.settle({secret: args.secret})
+              
+            } else {
+              forward.accept({})
+            }
+          });
+
+          sub.on('error', err => {
+            console.log(err)
```
Comment (alexbosworth) [inline offchain/sign_ghost_payment_request.js:218]:
```suggestion
            logger.error({err});
```

File: offchain/sign_ghost_payment_request.js [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+          const finished = (err, res) => {
+            sub.removeAllListeners();
+
+            return cbk(err, res);
+          };
+          
+          sub.on('forward_request', forward => {
+            if (args.id === forward.hash) {
+              forward.settle({secret: args.secret})
```
Comment (alexbosworth) [inline offchain/sign_ghost_payment_request.js:210]: shouldn't it exit here

Comment (niteshbalusu11) [reply offchain/sign_ghost_payment_request.js:210]: There seems to be an issue where it's exiting too early after settling and the payment is failing so I didn't exit. Didn't know how to deal with it.

Comment (alexbosworth) [reply offchain/sign_ghost_payment_request.js:210]: interesting, maybe there is a problem in ln-service

**Pull request closed without merging**